src/containers/EditAnimalModal.js

```diff
@@ -116,6 +116,11 @@ class EditAnimalModal extends Component {
       let errors = this.state.errors;
       errors["name"] = 'Nombre no puede ser vacio';
       this.setState({ errors: errors });
+    } else if ((this.props.animal.species_id == "1" || this.props.animal.species_id == "2" || this.props.animal.species_id == "5" )
```

Comment:
Usa `let { species_id } = this.props.animal` para que este if te quede mucho mas corto y claro.
Usa comillas simples